spaceship/spec/tunes/members_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe Spaceship::Tunes::Members do
+  before { Spaceship::Tunes.login }
+  before { TunesStubbing.itc_stub_members }
+
+  describe "members" do
+    it "should return a list with members" do
+      members = Spaceship::Members.all
+      expect(members.length).to eq(3)
+    end
```

Comment:
Can you always add new lines after an `end` block unless there is another `end` block following?